src/main/java/org/jenkinsci/plugins/durabletask/ProcessLiveness.java

```diff
@@ -94,14 +103,49 @@ private static boolean _isAlive(VirtualChannel channel, int pid, Launcher launch
             this.pid = pid;
         }
         @Override public Boolean call() throws RuntimeException {
```

Comment:
Would be great to document possible exceptions in the `Liveness` Javadoc

Reply:
Done.